computing.md

```diff
@@ -0,0 +1,142 @@
+# Setting up your computer
+
+Most projects developed in the lab will require you to have 
+git, a terminal, and Python properly setup on your computer.
+Below are some instructions on how to do this.
+
+## Git and a terminal
+
+[Git](https://git-scm.com/) is what we use to collaborate on 
+projects, track a history of changes, and backup to the cloud
+(GitHub). 
+
+A terminal with a decent shell (e.g., bash) is the primary
+interface for using git and other command line utilities
+(LaTeX compilers, make, etc).
+
+### Windows
+
+You can get both git and a bash-enabled terminal with "Git Bash".
+Follow the [Software Carpentry setup instructions](https://carpentries.github.io/workshop-template/#shell)
+to get going.
+
+### Linux
+
+You should already have a terminal with bash (look for the "terminal" app).
+Git is often already installed as well or you can install it with
+your distributions package manager.
+
+On Ubuntu:
+
+```bash
+sudo apt-get install git
+```
+
+## Python
+
+> **DO NOT** download Python from https://www.python.org!
+
+The best way to get setup with Python for your project is 
+by getting a Python distribution that has the `conda` or `mamba`
+package managers. Anaconda itself (which you likely have from
+one of Leo's classes) is good for some cases but for your 
+project work it will likely cause more harm than good because
+it comes with too many libraries that we won't need (and is thus 
+a huge pain to update).
+
+The recommended distribution to install for Windows, Linux, and Mac 
+is [Mambaforge](https://github.com/conda-forge/miniforge#mambaforge).
+This is a minimalist version of Anaconda made by the 
+[conda-forge](https://conda-forge.org/) team. It comes only with Python
+and `mamba` (which we use to install other libraries).
+
+Follow the instructions below to download and setup Mambaforge.
+
+### Windows
+
+Download the [latest Mambaforge](https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Windows-x86_64.exe) 
+and run the installer. 
+
+This should create a `mambaforge` folder in your home directory
+(open Git Bash and run `cd` to get to your home).
+
+Now we need to make Git Bash aware of Mambaforge so that
+you have access to the `mamba` package manager.
+
+> Run the commands below on Git Bash.
+
+Run this command to append the initialization code your
+`.bashrc` file:
+
+```bash
+cat >> .bashrc <<EOF 
+# Mambaforge initialization code
+source ~/mambaforge/etc/profile.d/conda.sh
+conda activate
+EOF
+```
+
+After that, close Git Bash and open it again. 
+To test that your setup worked, run `python` and 
+check if the output looks something like the following:
+
```

Comment:
python did not work. The Microsoft store opened to download python.

Reply:
That means that the setup code above is not right. 😢